### PR TITLE
update github-action update versions.tf

### PR DESCRIPTION
**Pull request by theprashantyadav**
1.  Update License
       2. Update terraform.yml
       3. update version.tf in examples

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -174,3 +174,28 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2021 CloudDrove Inc.
```
Comment (bryantbiggs) [inline LICENSE:189]: eh?

**Comment by bryantbiggs** [comment]
Not sure what the intentions are here, but instead of trying to claim credit for an OSS module, maybe try contributing back to the community?

**Comment by bryantbiggs** [comment]
cc @antonbabenko

**Comment by antonbabenko** [comment]
https://github.com/clouddrove - the organization they were copying some of the modules by terraform-aws-modules and by cloudposse some time ago.

They are not the only company that does it, so there is nothing new or special.

Contributing to OSS modules?!? Why??? :)

**Pull request closed without merging**